### PR TITLE
shell-completion: replace "gdb" verb with "debug" for coredumpctl

### PR DESCRIPTION
**Pull request by jnpkrn**
Also offer --debugger option.  Both to reflect changes in v239.

### CODE REVIEW

**Comment by filbranden** [comment]
Actually, `coredumpctl gdb` is still valid, even though it's not listed in the `--help` text for coredumpctl anymore... Maybe it should still be allowed as a valid completion, not dropped altogether just yet?

Otherwise, looks great, thanks!

**Comment by jnpkrn** [comment]
I am aware it is still valid [1], but isn't it the point of both man pages and
shell completions to drive people to what's intended to be customary
at that very snapshot of the overall evolution?

And man page already does that...

[1] I was actually asking for having the debugger configurable: #9311

**Comment by keszybz** [comment]
The general policy is that we document/auto-complete/show-in-`--help` the same set of non-deprecated options, so this patch follows policy. Let's just merge it, since `gdb` is so short that anyone is unlikely to use tab-completion on it anyway.

File: shell-completion/zsh/_coredumpctl [hunk 1]
```diff
@@ -7,7 +7,7 @@ _coredumpctl_command(){
             'list:List available coredumps'
             'info:Show detailed information about one or more coredumps'
             'dump:Print coredump to stdout'
-            'gdb:Start gdb on a coredump'
+            'debug:Start debugger (gdb) on a coredump'
```
Comment (keszybz) [inline shell-completion/zsh/_coredumpctl:10]: Grammatical nitpick: this should be "a debugger" ("gdb" is a name, so no article, but "debugger" is a generic noun, so it needs one).

**Pull request merged**